lib/ipmi_hpmfwupg.c

```diff
@@ -48,6 +49,10 @@
 # include <config.h>
 #endif
 
+#ifndef MAX
+#define MAX(a,b) ((a) > (b) ? (a) : (b))
```

Comment:
There is already `__max()` and `__min()` macros in  `#include <ipmitool/helper.h>`. Please change to using that instead of defining `MAX` here.

Reply:
Thanks, updated.